lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1311,6 +1355,81 @@ instance (PassphraseMaxLength purpose, PassphraseMinLength purpose)
 instance ToJSON (ApiT (Passphrase purpose)) where
     toJSON = toJSON . toText . getApiT
 
+instance FromJSON ApiScript where
+    parseJSON =  withObject "Script" $ \o -> do
+        script' <- o .: "script"
+        (ApiScript . ApiT) <$> parseJSON script'
```

Comment:
I think this should work...

```suggestion
    parseJSON =  withObject "ApiScript" $ \o ->
        ApiScript . ApiT <$> o .: "script"
```

Reply:
indeed, done in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/595999169a24d1637760f2f14f02c4af9d3cc4ad